### DISCUSSION TITLE
Can't select formulation

### DISCUSSION THREAD

**Discussion by harrythmia** [Open-Systems-Pharmacology/Forum#967] [Questions & Problems]
I have faced an issue where I can't select my formulation after I changed it from dissolution to first order. How should I solve this issue? Is there any reason behind this? I am clueless as I am a complete newbie here. :D

Also, is there any way that I can import metabolizing enzyme variants into PK-Sim? Like what type of files does it support and how should I import it?

Thank you in advance.

Yours,
Harry
![github](https://user-images.githubusercontent.com/90299857/133187191-03fceaca-3ed3-4deb-862d-d0cd2a187d7e.PNG)

**Comment by tobiasK2001** [comment]
hm, strange. I can reproduce the issue for first and zero order formulation. All other formulation work and I cannot see why it should not work with zero or first order. @msevestre, @Yuri05 Can this be a bug?

**Reply by Yuri05** [reply]
https://github.com/Open-Systems-Pharmacology/Forum/issues/127 😉 
Feel free to create the feature request for this missing feature.
S. also https://github.com/Open-Systems-Pharmacology/PK-Sim/issues/1470

**Comment by Yuri05** [comment]
It's a missing feature. Currently Zero/First Order formulations cannot be combined with oral administrations.

As a workaround: approximate your first order formulation with the *table formulation*. For that:

1. Open your first order formulation, right click on the formulation **graph**, select "Export to Excel" and export your first order formulation to excel file:
<p align="center">
<img width="600" src="https://user-images.githubusercontent.com/25061876/133218182-4ca5d53d-bd3d-4383-be74-7d2922b8b016.png">
</p>

2. Create new **table** formulation and import it from the exported First Order formulation file:
<p align="center">
<img width="600" src="https://user-images.githubusercontent.com/25061876/133218495-913752e9-390d-4854-9a7f-924285661a3e.png">
</p>

3. When creating your simulation: select this table formulation instead of first order.
